app/models/user.rb

```diff
@@ -14,6 +14,9 @@ class User < ApplicationRecord
   accepts_nested_attributes_for :person, allow_destroy: true
   after_find :sync_user_email
 
+  has_many :favourite_rooms
```

Comment:
again, necessary? Otherwise, the correct dependent option would probably be `:destroy` imo